.github/workflows/lint.yml

```diff
@@ -0,0 +1,63 @@
+name: Code Linting
+
+on:
+  pull_request:
+    branches:
+      - master
+      - develop
```

Comment:
@tofumatt Shouldn't this be under `on:push:` rather than under `on:pull_request:`?